src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -172,10 +172,10 @@ xorGate i1 i2 o = [rise i1, rise i2] ~|~> rise o <> [fall i1, fall i2] ~|~> rise
                <> [rise i1, fall i2] ~|~> fall o <> [fall i1, rise i2] ~|~> fall o
 
 srLatch :: Eq a => a -> a -> a -> CircuitConcept a
-srLatch s r q = complexGate (Var s) (Var r) q
+srLatch s r q = complexGate (Var s) (Var r) q <> mutex s r
```

Comment:
Wait, I think `mutex` is too restrictive -- in this way you restrict the behaviour of inputs. You need `never` here, because this is just an assumption about the environment, but not a restriction. In this way, the verification tools will spot an error if inputs are not mutually exclusive.